util.hpp

```diff
@@ -21,7 +21,7 @@ namespace Sass {
   string comment_to_string(const string& text);
   string normalize_wspace(const string& str);
 
-  string quote(const string&, char q = 0);
+  string quote(const string&, char q = 0, bool flag = false);
```

Comment:
Didn't yet come around to look more into it. But it's the reason why I added WIP again. Again I don't really know if we need to move the code part that uses this flag to another function. Will have another look after work :smile: